go.mod

```diff
@@ -20,6 +20,8 @@ require (
 	volcano.sh/apis v1.2.0-k8s1.19.6
 )
 
+replace github.com/kubeflow/common v0.3.7 => github.com/gaocegege/common v0.0.0-20211108040227-6ddf8b59f7bf
```

Comment:
We need to fix the common first, then add the LabelSelector to the common.JobStatus.

Or the PR will be blocked.